src/splitInteger.test.js

```diff
@@ -1,21 +1,44 @@
-'use strict';
+"use strict";
 
-const splitInteger = require('./splitInteger');
+const splitInteger = require("./splitInteger");
 
-test(`should split a number into equal parts
+describe(`Split integer`, () => {
+  test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
+    expect(splitInteger(8, 1)).toEqual([8]);
+    expect(splitInteger(6, 2)).toEqual([3, 3]);
+    expect(splitInteger(17, 4)).toEqual([4, 4, 4, 5]);
+    expect(splitInteger(32, 6)).toEqual([5, 5, 5, 5, 6, 6]);
+  });
 
-});
-
-test(`should return a part equals to a value
+  test(`should return a part equals to a value
   when splitting into 1 part`, () => {
-
-});
-
-test('should sort parts ascending if they are not equal', () => {
-
-});
-
-test('should add zeros if value < numberOfParts', () => {
-
+    const isDifferenceValid = (array) => {
+      if (array.length === 0) {
+        return true;
+      }
+
+      const max = Math.max(array);
+      const min = Math.min(array);
+
+      return max - min <= 1;
+    };
+
+    expect(isDifferenceValid(splitInteger(17, 4))).toEqual(
+      isDifferenceValid(splitInteger(17, 4))
+    );
```

Comment:
The test here is not effectively testing the `isDifferenceValid` function. It compares the result of `isDifferenceValid(splitInteger(17, 4))` with itself, which will always be true. Consider testing the function with different inputs to ensure it behaves as expected.